packages/sdk/contractkit/README.md

```diff
@@ -23,7 +23,7 @@ npm install @celo/contractkit
 yarn add @celo/contractkit
 ```
 
-You will need node version `8.13.0` or higher. Node version 10 is recommended.
+You will need node version `v8.13.0` or higher. Node version v12.x is recommended.
```

Comment:
Do we know that this works on node 8?

Reply:
No, good point. Do you think it is worth verifying if the repo works with node 8? Or should we just change that to node 10, I know CI passes when run with node 10.

Reply:
Personally, I would vote to just support any node releases that are currently in LTS or maintenance - (https://nodejs.org/en/about/releases/), so actually I think we should bump to 12. Ideally we would get to 16 for active development and just support 12 and above. Even if we know it works on 10 today we don't have the testing infra to know that in future development

Reply:
Agreed, also spoke with @barbaraliau about this and we came to the same conclusion (without the infra we can only really support a single version) so we will stick to just 12 for now.